File: giorno1.py
#AREA RETTANGOLO
base = int(input("inserisci la base: "))
altezza = int(input("inserisci l'altezza: "))
area=(base*altezza)/2
print("L'area del rettangolo è:", area)


#SCAMBIO VALORI DI DUE VARIABILI
x = int(input("Inserisci x: "))
y = int(input("Inserisci y: "))

z = x
x = y
y = z

print("Dopo lo scambio: x =", x, ", y =", y)


#CONVERSIONE CELSIUS FAHRENHEIT  
celsius = float(input("Inserisci la temperatura in gradi Celsius: "))

fahrenheit = (celsius + 17.7) * (9/5)
print("La temperatura in gradi Fahrenheit è:", fahrenheit)

File: giorno1.js
//AREA RETTANGOLO
let base=prompt("inserisci la base: ")
let altezza=prompt("inserisci l'altezza: ")
let area=(base*altezza)/2
console.log("l'area del rettangolo è:" + area)


//SCAMBIO VALORI DI DUE VARIABILI
let x = parseInt(prompt("Inserisci x:"));
let y = parseInt(prompt("Inserisci y:"));

let z = x;
x = y;
y = z;

console.log("Dopo lo scambio: x = " + x + ", y = " + y);


//CONVERSIONE CELSIUS FAHRENHEIT  
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});
rl.question("Inserisci la temperatura in gradi Celsius: ", function(celsius) {
  let fahrenheit = (parseFloat(celsius) + 17.7) * (9/5);
  console.log("La temperatura in gradi Fahrenheit è: " + fahrenheit);
  rl.close();
});
